PID: 40928
honu.cs.washington.edu
Job started at Wed Aug 21 00:14:44 PDT 2019
[RESOURCES] There are too many total processes running: 46!
Running patch plausibility analysis under Java-7 on the patch generated by Kali for Bears::dungba88-libra-436514153-436524727::0
patch_plausibility.py script started at Wed Aug 21 00:15:14 PDT 2019
Previous HEAD position was 2972046ed Changes in the tests This commit is based on the source code from the following commit: http://github.com/traccar/traccar/commit/3aebd92cc3781b417ded6260d1300434b1418e7e The mentioned commit triggered the following Travis build: http://travis-ci.org/traccar/traccar/builds/273136493.
Switched to branch 'dungba88-libra-436514153-436524727'
Note: checking out 'fcce805ea502a1abb63f320b6d144a007eea16a4'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at fcce805ea Changes in the tests This commit is based on the source code from the following commit: http://github.com/dungba88/libra/commit/fbe5d265849641ee3e49f5ffe81db87534525859 The mentioned commit triggered the following Travis build: http://travis-ci.org/dungba88/libra/builds/436524727.
Reinitialized existing Git repository in /tmp/Kali_Bears_dungba88-libra_436514153-436524727/.git/
Checking patch src/main/java/org/joo/libra/pointer/VariablePredicate.java...
Applied patch src/main/java/org/joo/libra/pointer/VariablePredicate.java cleanly.
test has not returned a non-zero exit code but it has not executed any test case either!
Traceback (most recent call last):
  File "/scratch/rjust/critical_review_of_repair_them_all_data/plausibility/patch_plausibility.py", line 175, in <module>
    test, all_tests, failing_tests = utils.test(bugObj, CHECKOUT_DIR, java_home_dir, os.path.join(output_dir, "test.log"))
  File "/scratch/rjust/critical_review_of_repair_them_all_data/java_7_vs_java_8/utils.py", line 179, in test
    return ret, all_tests, failing_tests, trigger_tests
NameError: global name 'trigger_tests' is not defined
Checkout Bears_dungba88-libra_436514153-436524727
Apply patch ...
Compile Bears_dungba88-libra_436514153-436524727
Run tests on Bears_dungba88-libra_436514153-436524727
patch_plausibility.py has failed!
